daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -2094,6 +2107,30 @@ private[lf] object SBuiltin {
 
   }
 
+  private[speedy] final case class SBUpdateContractCache(
```

Comment:
Speedy on ledger builtin for populating the ledger cached contracts - currently this is *only* stub code!!